README.md

```diff
@@ -9,6 +9,7 @@ The _Password Manager Resources_ project exists so creators of password managers
 * **Password Rules**: Rules to generate compatible passwords with websites' particular requirements.
 * **Websites with Shared Credential Backends**: Groups of websites known to use the same credential backend, which can be used to enhance suggested credentials to sign in to websites.
 * **Change Password URLs**: To drive the adoption of strong passwords, it's useful to be able to take users directly to websites' change password pages.
+* **Websites that Append 2nd Factor to Password**: Some websites use a two-factor authentication scheme where the user must append a generated code to their password when signing in.
```

Comment:
I’m going to get very particular here. What you have is good, but I’m going to try to help you make it even more clear.

I think this could be phrased slightly better. Websites aren’t doing the appending (implied by that). And maybe we can make this even more clear by referring to “2FA codes”, or something like that. How about?

Websites Where 2FA Code is Appended to Password

Reply:
done